lib/tnt_api/request_handler.rb

```diff
@@ -34,7 +34,7 @@ def xml_builder(attrs={})
 
     def savon
       Savon.client(
-        adapter: :httpclient,
+        adapter: config.adapter || :httpclient,
```

Comment:
Where is the adapter get configured?

Reply:
https://github.com/BloomAndWild/tnt_api/pull/13/files#diff-6ce32bbda47090016fc97df7647976dcR4